pkg/reconciler/pipelinerun/pipelinerun.go

```diff
@@ -1102,7 +1102,7 @@ func getTaskrunWorkspaces(ctx context.Context, pr *v1beta1.PipelineRun, rpt *res
 		pipelineRunWorkspaces[binding.Name] = binding
 	}
 
-	if config.FromContextOrDefaults(ctx).FeatureFlags.EnableAPIFields == config.AlphaAPIFields {
+	if config.FromContextOrDefaults(ctx).FeatureFlags.EnableAPIFields != config.StableAPIFields {
```

Comment:
can we confirm that "not stable" always means "alpha" or "beta" -- wondering what happens if I set this to another value that's not "alpha", "beta" or "stable", hopefully it's a validation error 🤔

Reply:
Yes. 
If you see https://github.com/tektoncd/pipeline/blob/ba2eb8af4e626c0c010b1d020ecf9aac9a446eb1/pkg/apis/config/feature_flags.go#L219-L231, it will throw an error if its not `alpha, beta or stable`